dbt/adapters/databricks/api_client.py

```diff
@@ -88,8 +88,10 @@ def start(self, cluster_id: str) -> None:
 
         response = self.session.post("/start", json={"cluster_id": cluster_id})
         if response.status_code != 200:
-            raise DbtRuntimeError(f"Error starting terminated cluster.\n {response.content!r}")
-        logger.debug(f"Cluster start response={response}")
+            if self.status(cluster_id) not in ["RUNNING", "PENDING"]:
+                raise DbtRuntimeError(f"Error starting terminated cluster.\n {response.content!r}")
+            else:
+                logger.debug("Presuming race condition, waiting for cluster to start")
```

Comment:
When we query state after error, if it's pending or running it means the start failed due to race condition as another thread got the cluster started.

Reply:
should it be `info` rather than `debug`

Reply:
Anything that is info will show in the normal dbt output, so we are very conservative about what we log with info. 
 This is normal operation; if they are using multiple python models and the command api they will hit this, so I don't think this is worth bringing to the users' attention if it's running or pending.  For whatever reason, the cluster start API errors if you ask to start a cluster that is already in the process of starting.  If it's not running or pending, they'll get the full output from raising the error.